### PR TITLE
chore: introduce top-level Rakefile for easier task management across packages

### PR DESCRIPTION
**Pull request by jkodu**
> Note: This PR has a dependency on, all of the below:
> 	- https://github.com/dequelabs/axe-core-gems/pull/113
> 	- https://github.com/dequelabs/axe-core-gems/pull/119
> 	- https://github.com/dequelabs/axe-core-gems/pull/114
> 	- https://github.com/dequelabs/axe-core-gems/pull/115
> 	- https://github.com/dequelabs/axe-core-gems/pull/116
> 	- https://github.com/dequelabs/axe-core-gems/pull/117
> 	- https://github.com/dequelabs/axe-core-gems/pull/118

In order to be able to execute the top level Rakefile tasks that are introduced.
Given the lerna-esque/ monorepo directory structure followed, the top level task is purely a bunch of helper to invoke either scoped or all rake tasks of individual tasks.

Rest of the work in this PR is cleanup.

Closes issue:
- NA

## Reviewer checks

**Required fields, to be filled out by PR reviewer(s)**

- [x] Follows the commit message policy, appropriate for next version
- [x] Code is reviewed for accessibility
- [x] Code is reviewed for security

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -1,39 +1,100 @@
...
+    sh "cd #{pkgDir} && rake format"
   end
+end
 
-  desc "Display currently-installed and latest-available versions of axe-core lib"
-  task :status do
-    sh "npm view axe-core version && npm list axe-core"
+# clobber
+desc "clobber/clean all packages"
+task :clobber, [:pkg] do |t, args|
```
Comment (scurker) [inline Rakefile:79]: Is clobber a common term vs clean?

File: Rakefile [hunk 2]
```diff
@@ -1,39 +1,100 @@
...
+  else
+    pkgDir = "./packages/#{args[:pkg]}"
+    sh "cd #{pkgDir} && rake clobber"
   end
 end
+
+# format code
+desc "Format code using rubocop"
+task :format => [] do
+  sh "bundle exec rubocop --config rubocop.yml --auto-correct --fix-layout"
```
Comment (scurker) [inline Rakefile:99]: It looks like `rubocop.yml` was deleted. Is that intended?

**Comment by scurker** [comment]
Changes LGTM, just 2 questions.

File: Rakefile (hunk 2 above)
Comment (scurker) [reply Rakefile:99]: I see packages contain a `rubocop.yml` so I'm assuming that's the config that is supposed to be used

File: Rakefile (hunk 1 above)
Comment (jkodu) [reply Rakefile:79]: Actually this task does not need to be on top level. Removing it.

File: Rakefile (hunk 2 above)
Comment (jkodu) [reply Rakefile:99]: You are right. Copy pasta error this one. 
Individual packages have their own rubocop, and there is a helper task for that from here - https://github.com/dequelabs/axe-core-gems/pull/120/files#diff-52c976fc38ed2b4e3b1192f8a8e24cffR60

**Pull request merged**